cli/command/trust/signer_remove.go

```diff
@@ -78,24 +76,26 @@ func isLastSignerForReleases(roleWithSig data.Role, allRoles []client.RoleWithSi
 	return counter < releasesRoleWithSigs.Threshold, nil
 }
 
-func removeSingleSigner(cli command.Cli, repoName, signerName string, forceYes bool) error {
+// removeSingleSigner returns whether the signer has been removed during this operation and an error
+// Note: the signer not being removed doesn't necessarily raise an error (eg. User saying "No" to the confirmation prompt)
```

Comment:
Besides dropping the `bool` I would remove the `Note:` and add punctuation, something like:
```
removeSingleSigner attempts to remove a single signer.
The signer not being removed doesn't necessarily raise an error e.g. user choosing "No" when prompted for confirmation.
```

Reply:
will fix.